src/component/numberstable/NumbersTable.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import 'bulma/css/bulma.css';
+import './NumbersTable.css'
+
+const NumbersTable = (props)=>{
```

Comment:
Awesome :). Just need some styling to make the table breaks after a certain number unless you'll have a huge windows